Add mention of extended IProductSnapshot interfaces (#7680)
umbraco/UmbracoDocs: 17/umbraco-commerce/key-concepts/product-adapters.md

```

{% hint style="info" %}

> `IProductSnapshot` is the base model for what Umbraco Commerce needs. If you want to include images or measurements (if you for example need to do shipping based on weight), then there are other extended IProductSnapshot interfaces you can return.

Author: ```suggestion
`IProductSnapshot` is the base model for what Umbraco Commerce needs. If you want to include images or measurements (for example, if you need weight-based shipping), then there are other extended IProductSnapshot interfaces you can return.
```

> `IProductSnapshotWithImage`, `IProductSnapshotWithCategories` & `IProductSnapshotWithMeasurements` are all valid return types that allows you to enrich the product model.

Reviewer: Not sure if there are in fact more available interfaces - you probably know that better than me ;)
But how about we use a list here? 🤔 

```suggestion
Valid interfaces include, but are not limited to:

* `IProductSnapshotWithImage`
* `IProductSnapshotWithCategories`
* `IProductSnapshotWithMeasurements`
```

Author: These are all the ones I could find, which also makes mentioning it being not limited to them a bit misleading 🙂
